README.md

```diff
@@ -33,15 +33,15 @@ require 'vendor/autoload.php';
 use AltchaOrg\Altcha\ChallengeOptions;
 use AltchaOrg\Altcha\Altcha;
 
-$hmacKey = 'secret hmac key';
+$altcha = new Altcha('secret hmac key');
```

Comment:
New object-oriented usage of Altacha.

The idea is to having to "know" the key only once. This could enable tighter integration with frameworks, where Altcha could be initialized using a central key, e.g., `APP_SECRET` in Symfony.